--- a/Makefile
+++ b/Makefile
@@ -107,6 +107,7 @@ FFMPEG_CUSTOM_MUXERS:= \
 	mpegts \
 	ogg \
 	rtp \
+	rtsp \

@@ -135,7 +136,7 @@ FFMPEG_CUSTOM_PROTOCOLS:= \
-file http icecast pipe rtp tcp udp
+file http icecast pipe rtp rtsp tcp udp

@@ -259,9 +261,12 @@ FFMPEG_CONFIGURE:= \
 	--disable-vaapi \
 	--disable-vdpau \
-	--disable-outdevs
+	--disable-outdevs \
+	--enable-network \
+	--enable-avdevice \
+	--enable-indev=v4l2
